docs/server/settings.rst

```diff
@@ -273,6 +273,14 @@ Configuration settings for applications used by Pootle.
   Enable spam prevention through a captcha.
 
 
+.. setting:: POOTLE_REPORTS_MARK_FUNC
+
+``POOTLE_REPORTS_MARK_FUNC``
+  Default: ''
+
+  Provide mark data for user daily activity charts in admin Pootle reports
```

Comment:
I can't do it(
There is the answer on first question.
You can add marks on timeline if you provide right function.
params: username, start, end
return: array of {label: 'Label for mark', position: js_timestamp}